go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -244,22 +242,18 @@ func runOvnKube(ctx *cli.Context) error {
 
 		ovn.RegisterMetrics()
 
-		var nodeSelector *metav1.LabelSelector
 		var hybridOverlayClusterSubnets []config.CIDRNetworkEntry
 		if enableHybridOverlay {
 			hybridOverlayClusterSubnets, err = hocontroller.GetHybridOverlayClusterSubnets(ctx)
 			if err != nil {
 				return err
 			}
 
-			nodeSelector = &metav1.LabelSelector{
```

Comment:
@JacobTanenbaum so what we should actually do to make this all more generic is to add a new config option to ovnkube master for "--ignore-nodes=<label selector>" in the Kubernetes section. Then we don't need to do a bunch of custom stuff in my extensions branch that conflicts all the time.